test/controllers/journeys/income/IncomeNotCountedAsTurnoverControllerSpec.scala

```diff
@@ -40,122 +41,167 @@ class IncomeNotCountedAsTurnoverControllerSpec extends SpecBase with MockitoSuga
 
   def onwardRoute = Call("GET", "/foo")
 
-  val formProvider = new IncomeNotCountedAsTurnoverFormProvider()
-  val form = formProvider()
-  val taxYear = LocalDate.now().getYear
+  val formProvider       = new IncomeNotCountedAsTurnoverFormProvider()
+  val formWithIndividual = formProvider("individual")
+  val formWithAgent      = formProvider("agent")
 
-  lazy val incomeNotCountedAsTurnoverRoute = IncomeNotCountedAsTurnoverController.onPageLoad(taxYear, NormalMode).url
+  def incomeNotCountedAsTurnoverRoute(isPost: Boolean, mode: Mode): String =
```

Comment:
I appreciate it is just UTs, but best practice to not have a method with a boolean flag as an argument, I would recommend splitting this method in to two, one for handling POSTs and one for GETs.

Reply:
Made this change